spk/homeassistant/src/dsm-control.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/sh
+
+# Package
+PACKAGE="homeassistant"
+DNAME="Home Assistant"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PYTHON_DIR="/usr/local/python3"
+PYTHONPATH="${INSTALL_DIR}/share/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${INSTALL_DIR}/env/bin:${PYTHON_DIR}/bin:${PATH}"
+USER="homeassistant"
+PYTHON="${INSTALL_DIR}/env/bin/python3"
+
+CONFIG_DIR="${INSTALL_DIR}/etc"
+#clean logs are generated by home-assistant in the config dir
+#more verbose logs can be captured by using -v instead -b in the start-stop-daemon command and piping it to a file
+LOG_FILE="${CONFIG_DIR}/home-assistant.log"
+
+PID_DIR="${INSTALL_DIR}/var"
+PID_FILE="${PID_DIR}/homeassistant.pid"
+
+RUN_CMD="${PYTHON}"
+RUN_ARGS="-m ${PACKAGE} -c ${CONFIG_DIR} --open-ui"
+
+start_daemon ()
+{
+    start-stop-daemon -b -o -c ${USER} -S -u ${USER} -m -p ${PID_FILE} -x env PYTHONPATH=${PYTHONPATH} ${RUN_CMD} -- ${RUN_ARGS}
```

Comment:
With the comments I made above covered, the start command probably would look like this:
start-stop-daemon -b -o -c ${USER} -S -u ${USER} -m -p ${PID_FILE} -x PATH=${PATH} ${PYTHON} -- ${RUN_ARGS}